src/apps/media/src/app/components/MediaDetailsModal.js

```diff
@@ -150,7 +130,7 @@ export const MediaDetailsModal = memo(function MediaDetailsModal(props) {
         </div>
       </ModalContent>
       <ModalFooter className={shared.ModalFooter}>
-        <Button type="cancel" onClick={props.onClose}>
```

Comment:
Shouldn't the type be `cancel`? Is the design-system styling incorrect?

Reply:
fixed and made a pr change

<img width="145" alt="Screen Shot 2021-09-14 at 11 17 08 AM" src="https://user-images.githubusercontent.com/22800749/133311884-6b5dc173-6d9a-4f52-9f6e-0bd63483609c.png">